api/users/views.py

```diff
@@ -138,10 +153,10 @@ def get_default_odm_query(self):
 
     # overrides ListCreateAPIView
     def get_queryset(self):
-        # TODO: sort
-        query = self.get_query_from_request()
-        return User.find(query)
-
+        queryset = OSFUser.objects.filter(is_registered=True, date_disabled=None).order_by('-date_registered')
+        if self.request.version >= '2.3':
+            queryset = queryset.filter(merged_by__isnull=True)
```

Comment:
This was added for [this test here](https://github.com/CenterForOpenScience/osf.io/blob/develop/api_tests/users/views/test_user_list.py#L60) that checks for version 2.3. Since django-filter is based on the view's `get_queryset` method (rather than our filter's relying on `get_default_odm_query`) -- I added the same checks on `get_queryset` as [already existed](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/users/views.py#L135) on the odm query method